roles/users/tasks/update_keys.yml

```diff
@@ -1,9 +1,43 @@
 ---
-- name: Update authorized_keys for each user.
+- name: Merge managed_users and managed_admin users
+  set_fact:
+    pubkey_users: "{{ managed_users|list + managed_admin_users|list }}"
+
+# The following set_fact calls are apparently necessary to avoid using sudo on
+# localhost to clone the keys repo.
+- set_fact:
+    ansible_become_orig: "{{ ansible_become }}"
+
+- set_fact:
+    ansible_become: false
+
+- name: Clone the keys repo
```

Comment:
Well, I was thinking at the end of this ansible-playbook run.   Reason to: meh.  trash left behind, is all.